src/qtools/vis.cpp

```diff
@@ -160,25 +160,25 @@ void decompress_vis_lump(BSPLEAF* leafLump, byte* visLump, byte* output,
 	{
 		g_progress.tick();
 		dest = output + i * newVisRowSize;
+		if (lastUsedIdx >= 0)
+		{

```

Comment:
This should never happen unless there's an overflow at ~4 GB of VIS data. I think it would better to log an error here, like:
```
if (lastUsedIdx < 0) {
    logf("Overflow decompressing VIS lump!");
    return;
}
```

Reply:
added